README.md

```diff
@@ -164,21 +168,26 @@ We see that <img src="https://render.githubusercontent.com/render/math?math=%5Cw
 This ends up being a really useful fact for writing neater software. We can handle things like messy data (e.g. variable length time series) just during data preprocessing, without it complicating the model code. In [time_series_classification.py](/example/time_series_classification.py), the region we integrate over is given by `X.interval` as a standardised region to integrate over. In the example [irregular_data.py](/example/irregular_data.py), we use this to handle variable-length data.
 
 #### Different interpolation methods
+For a full breakdown into the interpolation schemes, see [Neural Controlled Differential Equations for Online Prediction Tasks](https://arxiv.org/pdf/2106.11028.pdf) where each interpolation scheme is scrutinised, and best practices are presented.
+
 In brief:
  * Do you need causality?
    * No: natural cubic splines.
    * Yes: Is your data multivariate with missing values?
-     * No: linear interpolation
      * Yes: rectilinear interpolation.
+     * No:
+       * Are you using an adaptive scheme?
+         * No: linear interpolation
+         * Yes: Hermite cubic splines with backwards differences
```

Comment:
Isn't the idea to stop using natural cubic splines entirely? In favour of hermite/linear/rectilinear for offline/d.online/c.online respectively?

Reply:
I would be in favour of that, but I didnt think that was the plan. Am I to mark all natural cubic spline examples as deprecated?

Reply:
Hmm. I'd certainly switch over all the documentation to reflect the trichotomy described in the paper.
For `natural_cubic_coeffs` I'd probably just add a note to its docstring mentioning that there are usually better alternatives now.

Reply:
Ok I've removed natural cubic splines entirely in place of Hermite cubic.